### ISSUE TITLE
question about  the model

### ISSUE THREAD

**Issue by xinli2008** [zhengchen1999/CAT#8]
Given an original image and a mask image for the model, can the model generate a repaired image to complete the mask region?
Best

**Comment by zhengchen1999** [comment]
I think the pre-trained models we provided cannot do this. But, you can train a new model according to the training setting of the inpainting task, and it may be possible.

**Comment by xinli2008** [comment]
thank you for your reply and i will try that!
